public/javascripts/global.js

```diff
@@ -31,6 +31,7 @@ var formatRoster = function(response){
       template = Handlebars.compile(source);
       console.log(team_deets);
 
+  $("#roster").empty();
```

Comment:
Can you make the roster ul a variable so that we don't have to use a selector here and then in the line below? Also you can chain the the empty and the append into one line.